app/config/index.js

```diff
@@ -142,6 +142,18 @@ function argv(configPath, appVersion) {
 				describe: 'Custom Client Certs password for corporate authentication (certificate must be in pkcs12 format)',
 				type: 'string'
 			},
+
+			ssoUser: {
+				default: '',
+				describe: 'User to use for SSO auth.',
+				type: 'string'
+			},
+			ssoPasswordCommand: {
+				default: '',
+				describe: 'Command to execute to retrieve password for SSO auth.',
+				type: 'string'
+			},
+
```

Comment:
I would remove the empty lines in 145 and 156 (maybe is just the editor showing them to me)